qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -100,7 +100,7 @@ class LinComb(CircuitGradient):
     }
 
     # pylint: disable=signature-differs, arguments-differ
-    def __init__(self, aux_meas_op: OperatorBase = Z):
+    def __init__(self, aux_meas_op: OperatorBase = Z):  # type: ignore[override]
```

Comment:
I don't think we should be adding any `type: ignore[override]` markers.  They're usually true typing failures, and mypy _should_ emit errors on them.

Reply:
I've added ignore override only wherever there already was corresponding pylint disable comment (I guessed someone verified it manually)